src/MusicalScore/Graphical/EngravingRules.ts

```diff
@@ -122,11 +122,13 @@ export class EngravingRules {
     private wedgeLineWidth: number;
     private tupletLineWidth: number;
     private lyricUnderscoreLineWidth: number;
+    private lyricUnderscoreLineWidthVexflow: number;
```

Comment:
We should not have here extra parameters for Vexflow. Please just set the paramters simply such, that it works for Vexflow. If we later need it, we might add a method setEngravingRulesForVexFlow() or make VexFlowEngravingRules or something like that.